### DISCUSSION TITLE
Importing via the import folder suddenly stopped working with no error messages

### DISCUSSION THREAD

**Discussion by Noobgonewild** [photoprism/photoprism#3685] [Q&A]
2023-08-30 13:16:25 INFO import completed in 13 s

2023-08-30 13:16:12 INFO import: imported 1 file

2023-08-30 13:16:12 INFO import: found no .ppignore file

2023-08-30 13:16:12 INFO moving files from import

There are plenty of photos in the import folder. It used to work fine then it suddenly stopped. The only change I can think of is me changing the time in bios, to match my truenas time with my computer time. I also updated the app of the latest version.
My mount folder is a NFS share.

**Comment by graciousgrey** [comment]
This looks like PhotoPrism cannot access the files in the import folder and therefore thinks there are no files to import. This could be related to a [permission](https://docs.photoprism.app/getting-started/troubleshooting/docker/#file-permissions) or connectivity issue.

You might get some more information when you turn on [debug logs](https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3).

**Reply by Noobgonewild** [reply]
Thanks for the reply, this is a strange issue, I got it to work following the next steps:
-unmounted and remounted share, tried to import, didn't work
-unmounted share, tried to import knowing it won't work;indeed it threw an error
-mounted the share again and it worked

This Is so strange
